torchci/components/TestInsights.tsx

```diff
@@ -1,6 +1,10 @@
 import { JobData } from "../lib/types";
 import React from "react";
 
+// The following jobs are not supported at the moment because neither the monitoring
+// script is running there at the moment (libtorch, bazel, android) nor the CI job
+// uploads to S3 due to connectivity issues (macos, rocm). TODO: Add support for macos
```

Comment:
but...if macos and rocm aren't yet supported, then shouldn't they be in this list?